src/commands/purge.ts

```diff
@@ -8,7 +8,10 @@ module.exports = new Command({
 	props: [{ name: 'number', required: false }],
 	async execute({ ia }) {
 
-		const num = ia.options.data[0].value ? Number.parseInt(ia.options.data[0].value.toString()) : 100
+		let num = 100
+		if (ia.options.data.length == 1){
+			num = Number.parseInt(ia.options.data[0].value!.toString())
+		}
```

Comment:
I kind of dislike this. Reassignable variable and unnecessary curly braces. Also you would use === in JS to make an exact comparison :)

Reply:
Yeah it's kinda Go'ish, set the default and then change it if needed..